Peer/ConnectorAPI/PRListAPI.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Peer.Domain;
+using Peer.Domain.Models;
+
+namespace Peer.ConnectorApi
+{
+    public class PRListApi
```

Comment:
PRListApi doesn't feel like a great name. I don't have a specific suggestion. Maybe shoot me a message and we can noodle on it